### ISSUE TITLE
Notification dismiss button height depends on invisible text

### ISSUE THREAD

**Issue by sowasred2012** [canonical/vanilla-framework#4034]
Related to: https://github.com/canonical-web-and-design/vanilla-framework/issues/4033

[The dismiss button on notifications](https://vanillaframework.io/docs/patterns/notification#buttons) could have its descriptive, hidden text sit within an `aria-label` attribute, but the button lacks a set height, so when text is removed, the button moves from its position in the notification:

![Peek 2021-10-12 16-41.gif](https://images.zenhubusercontent.com/5cd93aaf2406d87914830598/447e1800-2db9-4637-9a75-4370f52c7cfd)

It looks like it could be fixed by setting a height (i.e. `2rem`) but I'm not sure what other implications that has.
